dev/com.ibm.ws.webcontainer.security.jacc.1.5_fat/fat/src/com/ibm/ws/webcontainer/security/jacc15/fat/BasicAuthTest.java

```diff
@@ -53,29 +54,38 @@ public class BasicAuthTest extends CommonServletTestScenarios {
     // (hopefully) be reflected in the static references as well.
     private static TestName _name = new TestName();
 
-    @Rule
-    public TestName name = _name;
-
-    private static final TestConfiguration testConfig = new TestConfiguration(myServer, myLogClass, _name, appName);
-
-    @BeforeClass
-    public static void setUp() throws Exception {
-        //myServer.addInstalledAppForValidation(appName);
-        //LDAPUtils.addLDAPVariables(myServer);
+    @ClassRule
+    public static CheckpointRule checkpointRule = new CheckpointRule().setConsoleLogName(BasicAuthTest.class.getSimpleName()
+                                                                                         + ".log").setServerSetup(BasicAuthTest::serverSetUp).setServerStart(BasicAuthTest::serverStart).setServerTearDown(BasicAuthTest::serverTearDown);
```

Comment:
This would be more readable if you forced line breaks like this:
```suggestion
    public static CheckpointRule checkpointRule = new CheckpointRule() //
                    .setConsoleLogName(BasicAuthTest.class.getSimpleName() + ".log") //
                    .setServerSetup(BasicAuthTest::serverSetUp) //
                    .setServerStart(BasicAuthTest::serverStart) //
                    .setServerTearDown(BasicAuthTest::serverTearDown);
```